demo/requirements.txt

```diff
@@ -9,5 +9,5 @@ numpy==1.23.5
 pandas==1.5.3
 plotly==5.14.1
 regionmask==0.10.0
-git+https://${LEAF_PRIVATE_SOURCE_CREDENTIALS}@github.com/leaf-ai/unileaf-util
+git+https://${LEAF_PRIVATE_SOURCE_CREDENTIALS}@github.com/leaf-ai/unileaf-util@0.2.2
```

Comment:
For the moment this dependency is private. Next step will be to remove it, so that everyone can run the demo.